web/components/editor.tsx

```diff
@@ -72,6 +74,7 @@ export const editorExtensions = (simple = false): Extensions => [
   DisplayLink,
   DisplayMention,
   DisplayContractMention,
+  ReactComponent,
```

Comment:
i think we should have a more descriptive name for this extension, like TipTapMarketGrid or just MarketGrid

Reply:
Ah missed that, definitely